lib/xnu/reload.sh

```diff
@@ -1,7 +1,23 @@
 #!/bin/sh
-sudo kextunload /Library/Extensions/FridaKernelAgent.kext &>/dev/null
-sudo rm -rf /Library/Extensions/FridaKernelAgent.kext
-sudo cp -a ~/Library/Developer/Xcode/DerivedData/FridaKernelAgent-eaueaeplpgvooydjbumkfbqjrobj/Build/Products/Debug/FridaKernelAgent.kext /Library/Extensions/FridaKernelAgent.kext
-sudo chown -R root /Library/Extensions/FridaKernelAgent.kext
-sudo chgrp -R wheel /Library/Extensions/FridaKernelAgent.kext
-sudo kextload /Library/Extensions/FridaKernelAgent.kext
+
+KEXT_NAME=FridaKernelAgent.kext
+
+sudo kextunload "/Library/Extensions/$KEXT_NAME" &>/dev/null
+sudo rm -rf "/Library/Extensions/$KEXT_NAME"
+
+BUILD_DIR="$(mktemp -d)"
```

Comment:
Guess we should clean this up afterwards?

Reply:
but it's a temp dir...

Reply:
I know, just don't like leaving stuff there. (OCD :) )

Reply:
```

Reply:
Perhaps adding this right after would do the trick:
```
function cleanup {
  rm -rf "$BUILD_DIR"
}
trap cleanup EXIT
```